### PR TITLE
Added multiple scripts and example links to Copernicus Browser

### PR DESCRIPTION
**Pull request by jonasViehweger**
This adds multiple scripts and example links to Copernicus Browser for the top 10 most popular scripts on custom scripts.

Most of the scripts were reworked to all follow a similar schema. 

Some of the Copernicus browser example links are likely not working yet, since the evalscript was pointed to one that will only start existing with this merge request. This was done because the length of evalscripts is more restricted for the Copernicus link shortening. 

If this actually works how I did it has to be checked after this was merged.  

You don't need to run the scripts but please visually check if:

- Script outputs are arrays
- Number of bands correspond to output
- All bands which are used are also in setup

The changes in the layout file are to open a script tab immediately even if something else on the page takes a while longer to load.

### CODE REVIEW

File: sentinel-1/radar_vegetation_index/raw.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Radar vegetation index for Sentinel-1 (RVI4S1) SAR data
+// Institute: MRSLab, Indian Institute of Technology Bombay, India
+// Data requirements: Sentinel-1 GRD data
+function setup() {
+    return {
+        input: ["VV", "VH", "dataMask"],
```
Comment (chorng) [inline sentinel-1/radar_vegetation_index/raw.js:6]: dataMask was not used in the script. We can remove it from the input.

File: sentinel-2/evi/script.js [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
+   [0.35, 0x609635],
+   [0.4, 0x4f892d],
+   [0.45, 0x3f7c23],
+   [0.5, 0x306d1c],
+   [0.55, 0x216011],
+   [0.6, 0x0f540a],
+   [1, 0x004400],
+];
 
-if (EVI<-1.1) return [0,0,0];
```
Comment (chorng) [inline sentinel-2/evi/script.js:7]: There was a discussion about using visualiser or conditional statement in the past with Monja and Anja and they decided to go with conditional statement as it was eaiser for users to understand and adapt (see [here](https://git.sinergise.com/sh-vas/sh-vas/-/issues/1276#note_536205)).

But I do think it's better to use a visualiser. Maybe we can raise this again in the eob channel or cdse channel?

File: sentinel-2/markuse_fire/cdse.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+// QuickFire V1.0.0 by Pierre Markuse (https://twitter.com/Pierre_Markuse)
+// Adjusted for use in the Copernicus Browser (https://dataspace.copernicus.eu/browser/)
+// CC BY 4.0 International (https://creativecommons.org/licenses/by/4.0/)
+
+// Copernicus Browser does not have the band CLP, this was replaced with the isCloud() function
+// but do try to turn off cloudAvoidance if results aren't as expected.
+
+function setup() {
+    return {
+        input: ["B01", "B02", "B03", "B04", "B08", "B8A", "B11", "B12", "dataMask"],
```
Comment (chorng) [inline sentinel-2/markuse_fire/cdse.js:11]: B01 was not used and could be removed from the input

File: sentinel-2/ndmi/eob.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    };
+}
+
+function evaluatePixel(samples) {
+    let val = index(samples.B8A, samples.B11);
+    // The library for tiffs works well only if there is only one channel returned.
+    // So we encode the "no data" as NaN here and ignore NaNs on frontend.
+    const indexVal = samples.dataMask === 1 ? val : NaN;
+    return {
+        default: [...viz.process(val), samples.dataMask],
```
Comment (chorng) [inline sentinel-2/ndmi/eob.js:31]: Maybe use `imgVals = viz.process(val);` and `imgVals.concat(samples.dataMask)` to keep consistency?

File: sentinel-2/ndmi/script.js [hunk 5]
```diff
@@ -1,54 +1,23 @@
...
+       input: ["B8A", "B11", "dataMask"],
+       output: { bands: 4 }
+   };
+}
 
-   return [r, g, b];
-  
+function evaluatePixel(samples) {
+   let val = index(samples.B8A, samples.B11);
+   return [...viz.process(val), samples.dataMask];
```
Comment (chorng) [inline sentinel-2/ndmi/script.js:22]: Same as the comment for `ndmi/eob.js`

File: sentinel-2/ndwi/eob.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    [-0.8, 0x008000],
+    [0, 0xFFFFFF],
+    [0.8, 0x0000CC]
+];
+
+let viz = new ColorRampVisualizer(ramp);
+
+function setup() {
+    return {
+        input: ["B03", "B08", "SCL", "dataMask"],
```
Comment (chorng) [inline sentinel-2/ndwi/eob.js:13]: I guess B04 is missing and SCL is not needed?

File: sentinel-2/ndwi/eob.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    // The library for tiffs works well only if there is only one channel returned.
+    // So we encode the "no data" as NaN here and ignore NaNs on frontend.
+    const indexVal = samples.dataMask === 1 ? val : NaN;
+
+    const imgVals = [...viz.process(val), samples.dataMask];
+
+    return {
+        default: imgVals,
+        index: [indexVal],
+        eobrowserStats: [val, isCloud(samples.SCL) ? 1 : 0],
```
Comment (chorng) [inline sentinel-2/ndwi/eob.js:34]: `isCloud(samples)`

File: sentinel-2/ndwi/eob.js [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    };
+}
+
+function evaluatePixel(samples) {
+    let val = index(samples.B03, samples.B08);
+    // The library for tiffs works well only if there is only one channel returned.
+    // So we encode the "no data" as NaN here and ignore NaNs on frontend.
+    const indexVal = samples.dataMask === 1 ? val : NaN;
+
+    const imgVals = [...viz.process(val), samples.dataMask];
```
Comment (chorng) [inline sentinel-2/ndwi/eob.js:29]: Let's keep this consistent and use `imgVals = viz.process(val)` and `imgVals.concat(samples.dataMask)`

File: sentinel-2/ndwi/script.js [hunk 9]
```diff
@@ -1,39 +1,20 @@
-//VERSION=3
-//This script was converted from v1 to v3 using the converter API
+//VERSION 3
+function setup() {
+  return {
+    input: ["B03", "B08"],
```
Comment (chorng) [inline sentinel-2/ndwi/script.js:4]: Let's add dataMasks here and return `imgVals.concat(samples.dataMask)` as other `script.js` did

File: sentinel-2/true_color/README.md [hunk 10]
```diff
@@ -8,9 +8,9 @@ nav_exclude: true
 ---
 
 
-## Evaluate and visualize
- - [Sentinel Playground](https://apps.sentinel-hub.com/sentinel-playground/?source=S2&lat=41.9027835&lng=12.496365500000024&zoom=12&evalscripturl=https://raw.githubusercontent.com/sentinel-hub/customScripts/master/sentinel-2/true_color/script.js){:target="_blank"}    
- - [EO Browser](https://apps.sentinel-hub.com/eo-browser/#lat=41.9&lng=12.5&zoom=10&datasource=Sentinel-2%20L1C&time=2017-10-08&preset=CUSTOM&layers=B01,B02,B03&evalscript=bGV0IGdhaW4gPSAyLjU7CnJldHVybiBbQjA0LCBCMDMsIEIwMl0ubWFwKGEgPT4gZ2FpbiAqIGEpOwo%3D){:target="_blank"}   
+## Evaluate and visualize  
+ - [EO Browser](https://apps.sentinel-hub.com/eo-browser/#lat=41.9&lng=12.5&zoom=10&datasource=Sentinel-2%20L1C&time=2017-10-08&preset=CUSTOM&layers=B01,B02,B03&evalscript=bGV0IGdhaW4gPSAyLjU7CnJldHVybiBbQjA0LCBCMDMsIEIwMl0ubWFwKGEgPT4gZ2FpbiAqIGEpOwo%3D){:target="_blank"} 
+ - [Copernicus Browser](https://link.dataspace.copernicus.eu/9ju)
```
Comment (chorng) [inline sentinel-2/true_color/README.md:13]: Shall we add `{:target="_blank"} ` for Copernicus Browser as well?

File: sentinel-2/evi/script.js (hunk 2 above)
Comment (jonasViehweger) [reply sentinel-2/evi/script.js:7]: I think it's time to move on from those bulky if elseif construct. Just from a maintainability standpoint

**Pull request merged**